druid/src/widget/label.rs

```diff
@@ -195,14 +191,14 @@ impl<T: Data> Label<T> {
         self.font = font.into();
     }
 
-    fn get_layout(&mut self, t: &mut PietText, env: &Env) -> PietTextLayout {
+    fn get_layout(&mut self, t: &mut PietText, env: &Env, bounds: &Size) -> PietTextLayout {
         let font_name = self.font.resolve(env);
         let font_size = self.size.resolve(env);
 
         // TODO: caching of both the format and the layout
```

Comment:
We should cache the text layout only between consecutive `Widget::layout` and `Widget::paint` calls, not in further scope, is that correct?

Reply:
Related, but since I can't comment on non-modified lines: in `Label::update`, condition `!old_data.same(data)` is wrong, is it not? `data` can stay the same, but we need to request layout even if the environment changes.